frontend/javascripts/oxalis/controller/scene_controller.js

```diff
@@ -362,13 +369,13 @@ class SceneController {
         }
       }
     } else {
-      const { tdViewDisplayPlanes } = Store.getState().userConfiguration;
       for (const planeId of OrthoViewValuesWithoutTDView) {
         const pos = getPosition(Store.getState().flycam);
         this.planes[planeId].setPosition(pos);
         this.planes[planeId].setGrayCrosshairColor();
-        this.planes[planeId].setVisible(true);
-        this.planes[planeId].plane.visible = this.isPlaneVisible[planeId] && tdViewDisplayPlanes;
+        this.planes[planeId].setVisible(tdViewDisplayPlanes !== TDViewDisplayModeEnum.NONE);
+        this.planes[planeId].plane.visible =
+          this.isPlaneVisible[planeId] && tdViewDisplayPlanes === TDViewDisplayModeEnum.DATA;
```

Comment:
I know it's been like this before, but could you extend the `setVisible` interface to something like `setVisible(isVisible: boolean, isDataVisible?: boolean)`. Then, the calling code would be:

```js
this.planes[planeId].setVisible(
  tdViewDisplayPlanes !== TDViewDisplayModeEnum.NONE,
  this.isPlaneVisible[planeId] && tdViewDisplayPlanes === TDViewDisplayModeEnum.DATA
)
```

(or something similar)

That's still a bit convoluted but would read better imo than calling a setter and then changing an internal property immediately afterwards.